blockstack/blockstack-app
# Fix (app): default network

This PR
* uses a network object when creating an `RPCClient`.
* defines a default network that is used as fallback if no network is provided by the user
* fixes #646 
* fixes #645 

To test #645: 
1. run `yarn dev` and 
1. make a contract call and 
1. inspect the calls to transfer endpoint, it should be `stacks-node-api.blockstack.org`, not testnet-`master.blockstack.org`

To test #646 
1. set network.coreApiUrl to  `stacks-node-api.krypton.blockstack.org` in test-app and
1. run `yarn dev` and
1. make a contract call and inspect the network calls, all should be `stacks-node-api-krypton.blockstack.org`, not testnet-`master.blockstack.org`, nor `stacks-node-api.blockstack.org`.
